Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -218,7 +218,7 @@ - (void)setup {
     self.isSelectingWholeMonth = NO;
     self.dateFormatter = [NSDateFormatter new];
     self.dateFormatter.locale = self.locale;
-    self.dateFormatter.dateStyle = NSDateFormatterLongStyle;
```

Comment:
This changed the A11y label from
"May 5, 2023" to "Friday, May 5, 2023"

This was done to make it easier to explore by touch with a screen reader